exercises/accumulate/accumulate_test.go

```diff
@@ -37,10 +37,10 @@ func TestAccumulate(t *testing.T) {
 
 func BenchmarkAccumulate(b *testing.B) {
 	b.StopTimer()
-	for _, test := range tests {
+	for i := 0; i < b.N; i++ {
 		b.StartTimer()
 
-		for i := 0; i < b.N; i++ {
+		for _, test := range tests {
```

Comment:
Since we want to get benchmark for `Accumulate` function call, should `StartTimer` and `StopTimer` surround each function call rather the entire tests?

Reply:
I wouldn't think so since the test ranging is consistent and would not add a variable amount of time to each benchmark? These are rough relative measures? I could be incorrect.

Reply:
I fairly sure the timer manipulation is used to remove noticeable overhead from a test routine. Like if some heavy initialization or resetting was necessary between the benched function calls.

Reply:
In benchmarks where no "overhead code" is necessary, as in this one and others, it is perfectly fine to omit all the StopTimer and StartTimer calls.such as [prime-factors](https://github.com/exercism/go/blob/master/exercises/prime-factors/prime_factors_test.go#L22-L28). But it is ok as it stands.

Reply:
I'm not sure about timers in most of the cases. I think we can omit them in almost every test.

Reply:
I am also in the wagon to remove here since there is no big startup cost.